packages/popover/test/a11y.test.js

```diff
@@ -369,107 +362,65 @@ describe('a11y', () => {
   });
 
   describe('Tab order', () => {
-    ['default', 'focusElement'].forEach((suite) => {
```

Comment:
Removed dedicated suite for `focusElement` as we no longer have logic for moving focus programmatically.